src/sidebar.jsx

```diff
@@ -335,6 +339,17 @@ const DropdownWithKebab = ({
             },
             title: _("Rename")
         },
+        ...(selected.length === 1 && selected[0].type === "file" && allowedtoRead(selected[0], user))
+
+            ? [
+                { type: "divider" },
+                {
+                    id: "download-item",
+                    onClick: () => downloadFile(currentPath, selected[0]),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-175-20240126-132259-b04878fd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)